tests/Unit/ApparentHorizons/Test_Tags.cpp

```diff
@@ -39,6 +40,10 @@ auto create_strahlkorper_y11(const double y11_amplitude, const double radius,
   return Strahlkorper<Frame::Inertial>(coefs, strahlkorper_sphere);
 }
 
+void test_fastflow_tag() {
```

Comment:
Could be cleaner to just inline this check in the `SPECTRE_TEST_CASE` below, instead of adding this function (up to you).